torchgeo/datamodules/ssl4eo_benchmark.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SSL4EO datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+
+from ..datasets import SSL4EOLBenchmark
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class SSL4EOLBenchmarkDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the SSL4EO-L dataset.
```

Comment:
```suggestion
    """LightningDataModule implementation for the SSL4EO-L Benchmark dataset.
```
If this doesn't fit on one line we'll have to get creative to satisfy pydocstyle